hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
I agree that the alternatives that you propose are better. In my opinion the worst offender is `beam_energy`. However, we should only rename things when strictly necessary because any change break scripts that use the old syntax.